bundles/io/org.eclipse.smarthome.io.net.security.test/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Files created during test runs
```

Comment:
I try my best to clean those files up. But if an exception happens in tearDown, this might not happen. To be on the safe site I have added those to gitignore as well.

Reply:
The TemporaryFolder should also disappear if an exception is thrown, if I remember correctly.

Reply:
TemporaryFolder looks good, I'm going to use that